ted_data_eu/services/etl_pipelines/potential_customers_etl_pipeline.py

```diff
@@ -0,0 +1,68 @@
+from datetime import datetime
+from typing import Dict
+
+import numpy as np
+from pymongo import MongoClient
+from ted_sws.data_manager.adapters.triple_store import FusekiAdapter, TripleStoreABC
+
+from ted_data_eu import config
+from ted_data_eu.adapters.etl_pipeline_abc import ETLPipelineABC
+from ted_data_eu.services.currency_convertor import convert_currency
+
+DEFAULT_FUSEKI_DATASET_NAME = 'ted_data_dataset'  # TODO: temporary while not put in config resolver
+TED_DATA_MONGODB_DATABASE_NAME = 'ted_analytics'  # TODO: temporary while not put in config resolver
+
+
+class PotentialCustomersETLPipeline(ETLPipelineABC):
+
+    def __init__(self, triple_store: TripleStoreABC = None, mongo_client: MongoClient = None, query_limit: int = None):
+        self.triple_store = triple_store if triple_store else FusekiAdapter(host=config.FUSEKI_ADMIN_HOST,
```

Comment:
replace FusekiAdapter with GraphDBAdapter, for this you need to merge code from main.